VSTS/Tasks/SonarQubeScannerMsBuildBegin/SonarQubeHelper.ps1

```diff
@@ -0,0 +1,334 @@
+# When passing arguments to a process, the quotes need to be doubled and   
```

Comment:
@bgavrilMS I have actually copied it in both tasks (SonarQubeScannerMsBuildBegin and SonarQubeScannerMsBuildEnd folders) for now. I will make some optimizations to copy it in both tasks using a powershell script later on.